scripts/box

```diff
@@ -177,14 +177,30 @@ We URGE you to migrate to a supported release if/while you still have the chance
 
 function _upgrade() {
     arg=("$@")
-    packages=("${arg[@]:1}")
+    if [[ "${arg[*]}" == *--all* ]]; then
+        package_string="$(find /install -type f -name .*.lock | awk -F. '{print $2}')"
+        blacklist=("rtorrent" "qbittorrent" "deluge" "libtorrent" "panel" "transmission" "curl" "nginx")
+        for blocker in "${blacklist[@]}"; do
+            package_string="$(echo "$package_string" | sed "/$blocker/d")"
+        done
+        echo_info "Upgrading all packages, EXCEPT torrent clients and packages with alternative upgrade methods (apt/built-in)\nAttempting packages: '${package_string/$'\n'/, }'"
```

Comment:
Should be `${package_string//$'\n'/, }`.

Reply:
Good catch, thanks